var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -30,9 +30,13 @@ class Dealii(Package):
     """C++ software library providing well-documented tools to build finite
     element codes for a broad variety of PDEs."""
     homepage = "https://www.dealii.org"
-    url      = "https://github.com/dealii/dealii/releases/download/v8.4.0/dealii-8.4.0.tar.gz"
+    url      = "https://github.com/dealii/dealii/releases/download/v8.4.1/dealii-8.4.1.tar.gz"
```

Comment:
I tried 8.3.0 and it picked the version correctly and downloaded the file. I halted the compilation though as it takes a long time to finish.